node/src/chain_spec/oak.rs

```diff
@@ -188,6 +190,105 @@ pub fn oak_staging() -> ChainSpec {
 	)
 }
 
+pub fn oak_live() -> ChainSpec {
+	// Give your base currency a unit name and decimal places
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), TOKEN_SYMBOL.into());
+	properties.insert("tokenDecimals".into(), TOKEN_DECIMALS.into());
+	properties.insert("ss58Format".into(), SS_58_FORMAT.into());
+
+	ChainSpec::from_genesis(
+		// Name
+		"OAK",
+		// ID
+		"oak",
+		ChainType::Live,
+		move || {
+			let allocation_json =
+				&include_bytes!("../../../distribution/oak_alloc.json")[..];
+			let initial_allocation: Vec<(AccountId, Balance)> =
+				serde_json::from_slice(allocation_json).unwrap();
+			const ALLOC_TOKENS_TOTAL: u128 = DOLLAR * 1_000_000_000;
```

Comment:
the initial alloc is much lower right?

Reply:
yah, waiting for vesting and alloc to come through